src/commands/bundle.ts

```diff
@@ -0,0 +1,76 @@
+import { Flags } from '@oclif/core';
+import { Example } from '@oclif/core/lib/interfaces';
+import Command from '../base';
+import bundle from '@asyncapi/bundler';
+import { promises } from 'fs';
+import path from 'path';
+import { load } from '../models/SpecificationFile';
+
+const { writeFile } = promises;
+
+export default class Bundle extends Command {
+  static description = 'Bundle multiple asyncapi files together.';
+  static strict = false;
+
+  static examples: Example[] = [
+    'asyncapi bundle ./asyncapi.yaml > final-asyncapi.yaml',
+    'asyncapi bundle ./asyncapi.yaml --output final-asyncapi.yaml',
+    'asyncapi bundle ./asyncapi.yaml ./features.yaml --reference-into-components',
+    'asyncapi bundle ./asyncapi.yaml ./features.yaml --base ./asyncapi.yaml --reference-into-components'
+  ];
+
+  static flags = {
+    help: Flags.help({ char: 'h' }),
+    output: Flags.string({ char: 'o', description: 'The output file name. Omitting this flag will create a main.yaml.' }),
```

Comment:
`Omitting this flag will create a main.yaml` this is not true anymore right?

Reply:
Yeah it is not true, will update the flag